rdkit/Chem/AllChem.py

```diff
@@ -264,6 +264,8 @@ def _combiEnumerator(items, depth=0):
   ProductEduct = namedtuple('ProductEduct', 'products,educts')
   for chains in _combiEnumerator(sidechainSets):
     prodSets = reaction.RunReactants(chains)
+    for prods in prodSets:
+      yield prods
       if returnReactants:
         yield ProductEduct(prods, chains)
       else:
```

Comment:
You now yield either prods twice (returnReactants is false) or prods/ProductEduct (returnReactants is True).